Assets/Scripts/UI/InventoryScrollBar.cs

```diff
@@ -38,7 +38,15 @@ public void NullInventoryBar()
 	void SetIcon( Image image, Sprite icon = null )
 	{
 		image.sprite = icon;
-		image.color = Color.white;
+
+		if( !icon )
+		{
+			image.color = Color.white - (Color)new Vector4( 0f, 0f, 0f, 1f );
```

Comment:
If you just want to change the alpha of a color we have a `SetAlpha()` extension method for `Color` that would make your intent here more clear.